app/views/dashboard/show.html.erb

```diff
@@ -1,4 +1,18 @@
 <h2> <%= @merchant.name %> </h2>
 
-<%= link_to "My Items", "/merchants/#{@merchant_id}/items" %>
-<%= link_to "My Invoices", "/merchants/#{@merchant_id}/invoices" %>
+<%= link_to "My Items", "/merchants/#{@merchant.id}/items" %>
+<%= link_to "My Invoices", "/merchants/#{@merchant.id}/invoices" %>
+
+<h3>Favorite Customers</h3>
+<% @top_customers.each do |customer| %>
+  <p> <%= customer.first_name %> <%= customer.last_name %>:
+    <%= customer.transaction_count %> purchases </p>
+<% end %>
+
+<section>
+<h3>Items Ready To Ship</h3>
+<% @items_to_ship.each do |item| %>
+  <p> <%= item.name %>- <%= link_to "Invoice ##{item.invoice_id}", "/merchants/#{@merchant.id}/invoices/#{item.invoice_id}" %>
+    <%= item.invoice_creation.strftime('%A, %B %d, %Y') %> </p>
```

Comment:
Nice! I haven't used the %A %B syntax before -- learned something new here

Reply:
I did too lol. I pulled it from Lawrence on that one.